Languages/Java/questions.json

```diff
@@ -65,12 +65,40 @@
     },
     {
       "id": 7,
-      "question":"Java programming is not statically-typed, means all variables need not first be declared before they can be used.",
+      "question": "Java programming is not statically-typed, means all variables need not first be declared before they can be used.",
       "details": "Every variable in java needs to be declared first before use, else it throws compilation error",
       "questionType": "bool",
       "trueOrFalse": false,
       "shortAns": "No",
       "tag": "Java"
+    },
+
+    {
+      "id": 8,
+      "question": " When an array is passed to a method, what does the method receive?",
+      "details": "When an array is passed to a method, a reference of the array is received by the method",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "The reference of the array",
+      "tag": "Java"
+    },
+    {
+      "id": 9,
+      "question": "compareTo() returns",
+      "details": "compareTo() returns an int value",
```

Comment:
please phrase the shortAns in the form of "it IS ____" 

And please give good syntax to all "question" key variables, usually ending with a question mark.

Reply:
ok sir